src/factory.md

```diff
@@ -1,7 +1,8 @@
 # Factory
 
-Factories define how to generate widgets from collections of data. They are used inside GTK as well, but Relm4 uses them a bit differently.
+Factories are a methodology to easily create an UI which gets automatically generated and updated based on a dynamic data collection.
```

Comment:
I don't think methodology is the right word in this context.

Maybe this is better:

```
Factories are used to generate widgets from collections of data. 
If you modify the collection of data, the widgets are generated and updated automatically.
Also, they make it simple to modify the data from the UI.

> GTK has factories, too, yet under the hood they are quite different from Relm4's factories. 
> In most cases, Relm4's factories will be the easier solution but if you need more control, you can always use [GTK's factories with gtk4-rs](https://gtk-rs.org/gtk4-rs/git/book/todo_app_1.html).
```